dev-spec.txt

```diff
@@ -5,6 +5,7 @@ python >=3.10
 cmocean
 esmf
 gsw
+mpas_tools >=0.36.0,<1.0.0
```

Comment:
I didn't add this to the `pyproject.toml` because it's only available on conda-forge and the `pyproject.toml` doesn't know about conda-forge, only PyPI.